### PR TITLE
Completed hashing functions

### PR DESCRIPTION
**Pull request by Siri-isha**

### CODE REVIEW

File: GroupAnagrams.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//T.C O(1)
```
Comment (ishika2741) [inline GroupAnagrams.java]: Check TC

File: IsomorphicStrings.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import java.util.Arrays;
+//T.C O(n)
+//S.C O(n)
```
Comment (ishika2741) [inline IsomorphicStrings.java]: Check SC

**Review by ishika2741 — COMMENTED** [review]
Well done!

**Comment by ishika2741** [comment]
Well done!

File: IsomorphicStrings.java (hunk 2 above)
Comment (Siri-isha) [reply IsomorphicStrings.java]: Updated SC

**Pull request closed without merging**